File: src/main/java/com/alluvial/mds/contract/SingleOrder.java
package com.alluvial.mds.contract;

import java.io.Serializable;

public class SingleOrder implements Serializable
{
	private static final long serialVersionUID = ContractHelper.svnRevToLong("$Rev: 168 $");

	public int SecurityId;
	public String	Security;
    public char BidOrAsk;
    public long SortKey;
    public int SortSubKey;
    public char Action;
    public double Price;
    public double Volume;
    public int OrderType;
    public short BrokerNo;
    public long OrderNo;
    public long UpdateTime;
    public long UpdateTimeNS;

    @Override
	public String toString() {
		return "SingleOrder [Action=" + Action + ", BidOrAsk=" + BidOrAsk
				+ ", BrokerNo=" + BrokerNo + ", OrderNo=" + OrderNo
				+ ", OrderType=" + OrderType + ", Price=" + Price
				+ ", Security=" + Security + ", SecurityId=" + SecurityId
				+ ", SortKey=" + SortKey + ", SortSubKey=" + SortSubKey
				+ ", UpdateTime=" + ContractHelper.dateFormat.format(UpdateTime) + ", UpdateTimeNS="
				+ UpdateTimeNS + ", Volume=" + Volume + "]";
	}
}

File: src/main/scala/com/alluvia/types/market/SingleOrder.scala
package com.alluvia.types.market


case class SingleOrder(var orderNo: Long,
                        var date: java.util.Date,
                 var security: String,
                 var price: Double,
                 var volume: Double,
                 var value: Double,
                 var askBefore: Double,
                 var bidBefore: Double,
var action: Char,
var bidOrAsk: Char,
var orderType: Int
                        )